recipes/bitserializer/all/conanfile.py

```diff
@@ -45,21 +50,38 @@ def _compilers_minimum_version(self):
             "apple-clang": "12",
         }
 
+    @property
+    def _is_header_only(self):
+        # All components of library are header-only except csv-archive
+        return not self.options.with_csv
+
+    def _patch_sources(self):
+        # Remove 'ryml' subdirectory from #include
+        replace_in_file(self, os.path.join(self.source_folder, "include/bitserializer/rapidyaml_archive.h"), "#include <ryml/", "#include <", strict=False)
```

Comment:
Is it something only applyiable on 0.50? If yes, I would ask to use a separated patch instead, which is more organized.

Reply:
It's common patch for all versions, actually it's optional for old versions, but required for current and for new.
Why it's optional for old versions of **BitSerializer** - they also has **YAML** support, but I've enabled it in the **Conan** only since version **0.50** (there is check in the `validate()` method), because previously there was not available **RapidYaml** package. I can't enable **YAML** for previous versions of **BitSerializer** because of breaking changes in the **RapidYaml API** (old version of **BitSerializer** does not work with latest version of **RapidYaml** which available in **Conan**).

Reply:
I've converted to draft, just would like to do more tests with combinations of options.

Reply:
Just commited one small fix - the patch should be applied regardless of the `_is_header_only` property.

Reply:
Thank you! Very descriptive!